### PR TITLE
Clean up / refactor per rubocop suggestions

### PR DESCRIPTION
**Pull request by janosrusiczki**
Feel free to squash my many commits :smile:

### CODE REVIEW

**Comment by janosrusiczki** [comment]
Just a heads up @BrunoSalerno - review and tell me what to change.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -39,58 +40,4 @@
     # `true` in RSpec 4.
     mocks.verify_partial_doubles = false
   end
-
```
Comment (matkoniecz) [inline spec/spec_helper.rb]: What is the point of removing this comment? This removal only makes potential customization of tests more complicated.

Comment (janosrusiczki) [reply spec/spec_helper.rb]: @matkoniecz When I ran `rubocop --auto-correct --safe` it double commented the block, the looks of which I didn't like so I removed it. But you have a valid point, I restored it now and it's formatted better.

**Comment by BrunoSalerno** [comment]
@janosrusiczki Sorry for the delay! It looks good!

**Pull request merged**